src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -2257,6 +2257,8 @@ private String getDlnaOrgPnFlags(RendererConfiguration mediaRenderer, int locali
 				} else if (mime.equals("video/vnd.dlna.mpeg-tts")) {
 					// patters - on Sony BDP m2ts clips aren't listed without this
 					dlnaOrgPnFlags = "DLNA.ORG_PN=" + getMPEG_TS_EULocalizedValue(localizationValue, media.isHDVideo());
+				} else if (mime.equals(MP4_TYPEMIME)) {
+					dlnaOrgPnFlags = "*";
```

Comment:
@SubJunk In most cases for DLNA flags, the defaults work. If it is mentioned, it has to be correct. You can verify the same by removing these lines. 
The other thing to try is - copy the ffmpeg command line and generate an output from terminal in a file. Check if that works.